datadog-monitor-express-endpoint/variables.tf

```diff
@@ -44,7 +44,7 @@ variable "tags" {
 
 variable "message" {
   type = string
-  default = "@slack-figure-alerts"
+  default = "{{#is_alert}}@slack-platform-warnings{{/is_alert}} {{#is_alert_recovery}}@slack-platform-warnings{{/is_alert_recovery}}"
```

Comment:
Předpokládám že latenci API endpointů si taky chceme nechat spíš pro sebe. 🙂

Reply:
je tady specificky potreba to is_alert a is_alert_recovery? Zacalo to fungovat jinak?

Reply:
Asi ne, dodal jsem to tam spíš preventivně. Jinde to používáme, tak jsem to doplnil tam kde to nebylo. Explicitně chceme notifikace jen o vzniku a recovery alertu a ne o warningu a nějakých dalších stavech jako no_data, unknown . 🤔 Warningy u většiny monitorů ale ani nepoužíváme, tak je to zbytečný. Odstraním. 😄

Reply:
resp. v případě že tam je nastavený warning threshold, tak Rootly se notifikuje jen na alert a Slack vždycky: https://github.com/FigurePOS/terraform-modules/pull/38/commits/bca8aced8b0bbf7467b3059ef821355342cdb79d